waku.md

```diff
@@ -145,6 +134,7 @@ nonce = 8OCTET
 messages = 1*waku-envelope
 
 ; mail server / client specific
+; @TODO DO WE MOVE THIS TO MS SPEC?
```

Comment:
no imo, it belongs to this layer in terms of where logic is implemented (interface)

it can however be repeated in wms I suppose?